content_schemas/formats/organisation.jsonnet

```diff
@@ -140,7 +140,20 @@
                       type: "string",
                     },
                     image: {
-                      "$ref": "#/definitions/image",
+                      "oneOf": [
+                        {
+                          "$ref": "#/definitions/image",
+                        },
+                        {
+                          "type": "null"
+                        }
+                      ]
+                    },
+                    youtube_video_id: {
+                      type: [
+                        "string",
+                        "null",
```

Comment:
Similarly here I imagine we can just set this to a string if it exists otherwise not set the attribute, as it's also not a required property.